r/rsparkling/R/h2o_context.R

```diff
@@ -37,20 +36,22 @@ h2o_flow <- function(sc) {
 #'
 #' @param sc Object of type \code{spark_connection}.
 #' @param x A \code{spark_dataframe}.
-#' @param name The name of the H2OFrame
+#' @param name The name of the H2OFrame.
+#' @param strict_version_check (Optional) Setting this to FALSE does not cross check version of H2O and attempts to connect.
 #' @export
-as_h2o_frame <- function(sc, x, name=NULL) {
+as_h2o_frame <- function(sc, x, name=NULL, strict_version_check=TRUE) {
```

Comment:
I think this argument is necessary. Otherwise you limit a user to a specific version of H2O. Sparkling Water can handle any minor version AFAIK. Major version I am not sure. cc @mmalohlava